lib/schemas/operationContext.js

```diff
@@ -2,9 +2,9 @@
  * Copyright (c) 2019-2021 Digital Bazaar, Inc. All rights reserved.
  */
 'use strict';
-const sec = require('security-context');
+const {CONTEXT_URL} = require('webkms-context');
 
 module.exports = {
   type: 'string',
-  enum: [sec.constants.SECURITY_CONTEXT_V2_URL]
+  enum: [CONTEXT_URL]
```

Comment:
So here is a comment indicating that, at least in the case of a `GenerateKeyOperation` we need to have  the context for the key such as `Ed25519VerificationKey2020` `https://w3id.org/security/suites/ed25519-2020/v1`.

https://github.com/digitalbazaar/webkms-client/pull/24#discussion_r614308941

I have made this the case which is running afoul of this validation.

Before I proceed, I want to ensure that what is needed here is that we need to allow for:
- Array index 0 is always `webkms-context`
- Array index 1 is one of the context for one of the supported key types here: https://github.com/digitalbazaar/webkms-client/blob/56230b10d5de668d44dbdee1d6004d18c8dc445b/cryptoLd.js#L15-L19

Is all of this correct?

@dlongley @dmitrizagidulin

Reply:
@mattcollier, I looked at the schemas and it looks like we only need the key suite context when we're generating a key. Otherwise we don't. I think we should not use the `operationContext` in the `GenerateKeyOperation` schema and special case it there ... or take some other similar approach.

Reply:
I don't think the context array is order-preserving - right @dlongley? I vaguely recall we tried to fix the order for Verifiable Credentials, but that had more to do with systems that consumed the VCs as regular JSON.